src/Model/Behavior/SearchBehavior.php

```diff
@@ -62,10 +62,10 @@ public function findSearch(Query $query, array $options)
                 'to be nested under key "search" in find() options.'
             );
         }
-
+        
```

Comment:
You are introducing noise whitespace
Please make sure your IDE is adjusted to "trim right" on modified lines at least.